frappe/public/js/form_builder/components/AddFieldButton.vue

```diff
@@ -67,7 +67,7 @@ const fields = computed(() => {
 			return true;
 		})
 		.map((df) => {
-			let out = { label: df };
+			let out = { label: __(df), value: df };
```

Comment:
translate df.label is needed as it is part of the field property pane 
![image](https://github.com/frappe/frappe/assets/12823863/5d302b82-4c05-4210-bd30-53f4ec057839)